special_media/issuemapping-latvia

```diff
@@ -0,0 +1,70 @@
+[ministries]
+http://www.mk.gov.lv/
+http://www.mfa.gov.lv/en/
+http://www.km.gov.lv/en/
+http://www.sprk.gov.lv/
+http://www.izm.gov.lv/en/
+http://www.varam.gov.lv/lat/par_ministriju/
+http://www.fm.gov.lv/en/
+http://www.iem.gov.lv/lat/
+http://www.sam.gov.lv/sm/content/?cat=134
+https://www.em.gov.lv/en/
+
+[health]
+http://www.vm.gov.lv/en/
+http://www.vmnvd.gov.lv/en/about-nhs
+https://www.latvija.lv/epakalpojumi/ep113/apraksts
+http://www.lm.gov.lv/
+
+[security & oversight]
```

Comment:
ya it is a problem:

      File "/home/qq/T/trackmap/manyutils.py", line 228, in media_file_cleanings
        cleanurl = url_cleaner(line)
      File "/home/qq/T/trackmap/manyutils.py", line 103, in url_cleaner
        raise Exception("Invalid protocol in: %s" % line)
    Exception: Invalid protocol in: [security

the fact is: I can patch it easily, but if these string has to be visualized in html I've to bother also about escaping...